### DISCUSSION TITLE
Clone Pipelines

### DISCUSSION THREAD

**Discussion by marconimedeiros2** [dataplane-app/dataplane#367] [Ideas & feature requests]
A pipeline cloning feature to increase our productivity and reuse the pre-configuration of other pipelines.

It will give a lot of productivity with a feature of low complexity.

**Comment by saul-data** [comment]
Hi there @marconimedeiros2 you can do this in the Pipelines page by clicking on Manage > Duplicate. 

Here is a screenshot:
<img width="560" alt="image" src="https://user-images.githubusercontent.com/63714857/191314419-daa65858-5688-4d57-9592-600e6244745f.png">
